adafruit_ads1x15/ads1x15.py

```diff
@@ -66,15 +74,30 @@ class ADS1x15:
     :param int data_rate: The data rate for ADC conversion in samples per second.
                           Default value depends on the device.
     :param Mode mode: The conversion mode, defaults to `Mode.SINGLE`.
+    :param int comparator_queue_length: The number of successive conversions exceeding
+                          the comparator threshold before asserting ALERT/RDY pin.
+                          Defaults to 0 (comparator function disabled).
+    :param int comparator_low_threshold: Voltage limit under which comparator de-asserts
+                          ALERT/RDY pin. Must be lower than high threshold to use comparator
+                          function. Value is 12-bit, 2's complement stored in
+                          16-bit register where 4 LSBs are 0. Defaults to 0x8000 (decimal -32768).
```

Comment:
```suggestion
                          function. Value is a signed 17-bit integer.
```

Reply:
I just realized I forgot to update this comment in the higher level (ads1x15) class to be more generic, since the subclasses/chips handle this differently.
However, even for the ads1115, the conversion register (ADC data) and the threshold registers on the chip are only 16 bits total, whether or not you think of them as signed or unsigned, so there are only 2^16 (0 to 65535) possible values. So I don't understand why call it a 17-bit int? It would seem like python can handle it either way, as in the widths of ints don't have to be specified. But I agree that the comment should be correct so everyone understands.